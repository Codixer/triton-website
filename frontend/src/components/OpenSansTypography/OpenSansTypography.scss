:local(.openSansTypography) {
  font-family: "Open Sans", sans-serif;
}
