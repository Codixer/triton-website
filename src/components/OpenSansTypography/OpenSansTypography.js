import React, { Component } from "react";
import styles from "./OpenSansTypography.scss";
import { Typography } from "@rmwc/typography";
import "@material/typography/dist/mdc.typography.css";

class OpenSansTypography extends Component {
  render() {
    return <Typography className={styles.openSansTypography} {...this.props} />;
  }
}

export default OpenSansTypography;
