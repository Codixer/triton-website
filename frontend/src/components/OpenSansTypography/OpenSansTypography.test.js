import React from "react";
import OpenSansTypography from "./OpenSansTypography";
import renderer from "react-test-renderer";

it("<OpenSansTypography /> renders correctly", () => {
  const tree = renderer
    .create(<OpenSansTypography use="headline1">Test Typography</OpenSansTypography>)
    .toJSON();
  expect(tree).toMatchSnapshot();
});
