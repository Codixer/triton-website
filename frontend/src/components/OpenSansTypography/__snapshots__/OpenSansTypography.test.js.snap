// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<OpenSansTypography /> renders correctly 1`] = `
<span
  className="mdc-typography--headline1"
>
  Test Typography
</span>
`;
